demisto_sdk/commands/common/hook_validations/release_notes.py

```diff
@@ -77,6 +79,78 @@ def has_release_notes_been_filled_out(self):
                 return False
         return True
 
+    def is_docker_image_same_as_yml(self):
+        """
+        Iterates on all modified yaml files,
+        checking if the yaml is related to one of the sections in the RN and if there's a docker-image version update mentioned in the RN.
+        If so, make sure the versions match.
+
+        Return:
+            True if for all the modified yaml files, if there was a change in the docker image in the RN, it's the same version as the yaml.
+            Otherwise, return False and a release_notes_docker_image_not_match_yaml Error
+        """
+        release_notes_categories = self.get_categories_from_rn("\n" + self.latest_release_notes)
+        # renamed files will appear in the modified list as a tuple: (old path, new path)
+        modified_files_list = [file[1] if isinstance(file, tuple) else file for file in (self.modified_files or [])]
+        modified_yml_list = [file for file in modified_files_list if file.endswith('.yml')]
+        rn_file_name = self.release_notes_file_path[self.release_notes_file_path.rindex('/') + 1:]
+        error_list = []
+        for type, field in zip(['Integrations', 'Scripts'], ['display', 'name']):
+            if(type in release_notes_categories):
+                splited_release_notes_entities = self.get_entities_from_category("\n" + release_notes_categories.get(type))
+                for modified_yml_file in modified_yml_list:
+                    modified_yml_dict = get_ryaml(modified_yml_file) or {}
+                    if modified_yml_dict.get(field) in splited_release_notes_entities:
+                        docker_version = self.get_docker_version_from_rn(splited_release_notes_entities.get(modified_yml_dict.get(field)) + "\n")
+                        if docker_version and modified_yml_dict.get("script", {}).get("dockerimage") != docker_version:
+                            error_list.append({'name': modified_yml_dict.get(field),
+                                               'rn_version': docker_version,
+                                               'yml_version': modified_yml_dict.get("script", {}).get("dockerimage")})  # type:ignore
+        if len(error_list) > 0:
+            error_message, error_code = Errors.release_notes_docker_image_not_match_yaml(rn_file_name, error_list)
+            if self.handle_error(error_message, error_code, file_path=self.release_notes_file_path):
+                return False
+
+        return True
+
+    @staticmethod
+    def get_docker_version_from_rn(section):
+        """
+        Strips the docker image version from the relevant section in the RN if noted.
+        Args:
+            section : the section to search its docker image version note
+        Return:
+            str. The docker image version if exists. otherwiser, return None.
```

Comment:
```suggestion
    def get_docker_version_from_rn(section: str) -> str:
        """
        Extract the docker image version from the relevant section in the RN if mentioned.
        Args:
            section (str): the section to search its docker image version note
        Return:
            (str): The docker image version if exists, otherwise, return None.
```